Editor/ProjectAuditorSettings.cs

```diff
@@ -27,5 +27,10 @@ public class ProjectAuditorSettings : ScriptableObject
         /// Maximum size of all files in the StreamingAssets folder. Beyond that size we report an issue.
         /// </summary>
         public int StreamingAssetsFolderSizeLimit = 50;
+
+        /// <summary>
+        /// Maximum size when it is not necessary to enable Streaming Mipmaps. Beyond that size we report an issue.
+        /// </summary>
+        public int TextureStreamingMipmapsSizeLimit = 1000;
```

Comment:
At the moment we don't know what's the most appropriate threshold. For the time being, let's bump it to 4k so it only triggers on very large textures.